core/services/directrequestocr/drlistener.go

```diff
@@ -352,13 +356,22 @@ func (l *DRListener) handleOracleRequest(request *ocr2dr_oracle.OCR2DROracleOrac
 
 	ctx, cancel := l.getNewHandlerContext()
 	defer cancel()
-	run := pipeline.NewRun(*l.job.PipelineSpec, vars)
+
 	err := l.pluginORM.CreateRequest(request.RequestId, time.Now(), &request.Raw.TxHash, pg.WithParentCtx(ctx))
 	if err != nil {
 		l.logger.Errorw("failed to create a DB entry for new request", "requestID", formatRequestId(request.RequestId), "err", err)
 		return
 	}
 	l.markLogConsumed(lb, pg.WithParentCtx(ctx))
+
+	if uint32(len(request.Data)) > l.pluginConfig.MaxRequestSizeBytes {
+		errMsg := fmt.Sprintf("request too big (max %d bytes)", l.pluginConfig.MaxRequestSizeBytes)
```

Comment:
here you duplicate MaxRequestSizeBytes, in the errMsg and as Errorw param, maybe leave errMsg simple "request too big" then in Grafana we can easily filter these.